config/locales/en/general.yml

```diff
@@ -443,7 +443,7 @@ en:
       retired: Proposal retired by the author
       share: Share
       send_notification: Send notification
-      no_notifications: "This proposal has any notifications."
+      no_notifications: "This proposal doesn't have any notification."
```

Comment:
I was wondering if the correct text would be  `This proposal doesn't have any notification.` or `This proposal doesn't have any notifications.` I thought the plural form was the right one, but I'm not a native speaker. Could you confirm which one is correct?